### DISCUSSION TITLE
change color of YAxis or XAxis text

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [recharts/recharts#2994] [Q&A]
how can I change its color? I couldn't figure it out myself!

![usage](https://user-images.githubusercontent.com/42802922/193685050-c379d35a-0867-4958-95c6-25520f880886.png)

**Comment by jaytzanavar** [comment]
Hey man , Its not the most recommended practice but I ve changed the color of the axis through class css , with the attribute fill: color of your choice !important (for safety reasons :P )

**Comment by ckifer** [comment]
Hi

You can change it with the `fill` prop on the `tick` object prop of X or Y axis

```
<XAxis dataKey="name" tick={{ fill: "red" }}/>
<YAxis tick={{ fill: "blue" }} />
```

https://codesandbox.io/s/simple-line-chart-forked-wwpb8u?file=/src/App.tsx
